### PR TITLE
Move to go modules

### PR DESCRIPTION
**Pull request by hanzei**

### CODE REVIEW

File: server/issue.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/google/go-github/github"
+	"github.com/google/go-github/v28/github"
```
Comment (cpanato) [inline server/issue.go]: this is to lock the version?

Comment (hanzei) [reply server/issue.go]: Kind of. This is just how imports work with go modules. The version _has_ to be part of the module path. See https://github.com/golang/go/wiki/Modules#semantic-import-versioning and https://github.com/google/go-github/blob/master/go.mod#L1

Comment (cpanato) [reply server/issue.go]: 👍

**Comment by hanzei** [comment]
Follow up question: Do we still want to use the `vendor` folder? If so, I would alter the makefile to always use it.

**Pull request merged**